eng/versioning/pom_file_version_scanner.ps1

```diff
@@ -0,0 +1,438 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# Use case: This script verifies the following:
+# 1. There are no duplicate entries in any of the version_*.txt files
+# 2. There are no duplicate entries in the external_dependencies.txt file
+# 3. POM file verification across the enlistment which includes the following:
```

Comment:
nit: enlistment is old terminology in the git world we generally refer to it as a repo.

Reply:
Already changed, will be updated with the next commit.